codecarbon/emissions_tracker.py

```diff
@@ -416,21 +419,23 @@ def _init_output_methods(self, api_key):
             self.persistence_objs.append(HTTPOutput(self._emissions_endpoint))
 
         if self._save_to_api:
-            self._cc_api__out = CodeCarbonAPIOutput(
+            _cc_api__out = CodeCarbonAPIOutput(
                 endpoint_url=self._api_endpoint,
                 experiment_id=self._experiment_id,
                 api_key=api_key,
                 conf=self._conf,
             )
-            self.run_id = self._cc_api__out.run_id
-            self.persistence_objs.append(self._cc_api__out)
+            self.run_id = _cc_api__out.run_id
+            self.persistence_objs.append(_cc_api__out)
 
         else:
             self.run_id = uuid.uuid4()
 
         if self._save_to_prometheus:
-            self._cc_prometheus_out = PrometheusOutput(self._prometheus_url)
-            self.persistence_objs.append(self._cc_prometheus_out)
+            self.persistence_objs.append(PrometheusOutput(self._prometheus_url))
+
+        if self._save_to_logfire:
+            self.persistence_objs.append(LogfireOutput())
```

Comment:
`persistence_objs` seems highly related to the #458 composite output handling. Can we merge concepts in an elegant way ? Maybe @eledhwen would have an idea ?